codec-http/src/main/java/io/netty/handler/codec/http/websocketx/WebSocketServerProtocolHandshakeHandler.java

```diff
@@ -117,7 +117,7 @@ private boolean isWebSocketPath(FullHttpRequest req) {
         String websocketPath = serverConfig.websocketPath();
         String uri = req.uri();
         boolean checkStartUri = uri.startsWith(websocketPath);
-        boolean checkNextUri = checkNextUri(uri, websocketPath);
+        boolean checkNextUri = "/".equals(websocketPath) ? true : checkNextUri(uri, websocketPath);
```

Comment:
```suggestion
boolean checkNextUri = "/".equals(websocketPath) || checkNextUri(uri, websocketPath);
```
This can be simplified

Reply:
fixed